### ISSUE TITLE
That was very low of you and a message to all guests and potential posters

### ISSUE THREAD

**Issue by dmxt** [Entomy/Offbase-theme#2]
In last thread, you attempted to smear me by locking it with half-decade old screenshots, designed in 2013 and 2015, way before WCAG AA or AAA standards, way back when low-contrast was trending with all of those older tech monitors.

If you cannot take it like as if constructive criticism is _racist_, then I advise you to stop demanding any kind of constructive criticism until you grow up. You're insulting to all victims by watering down this term to this very, very low point. 

I'll do my part for citizens below.

**Message for everyone else who is considering to honor his request and post an issue about his work:** 

Are you considering to give him constructive criticism?

Don't bother. He can't take any of constructive criticism, he deem it as the same level as racism. He'll just lash out at you and do everything in his power to justify that he's right, including stalking your profile and digging up old irrelevant stuff because he's too insecure for real constructive criticism and attempt to smear out and discredit anybody who dared to give him one.

He also suggested all of you guys open your wallet and join his little patreon website for the dead cause since the project isn't being maintained. I won't imagine him being mad at you for giving him free money as long you keep your mouth shut and not trigger him out.

**Comment by Entomy** [comment]
If anyone is concerned about my handling of criticism, I encourage you to look at other projects, with far more reported issues. For example: [this extension](https://github.com/Entomy/Ada-vscode/issues?q=is%3Aissue+is%3Aclosed)

This guys comments have been the first I've ever needed to lock.
